src/understand-tidb/execution-framework.md

```diff
@@ -1 +1,41 @@
 # Execution Framework
```

Comment:
Maybe we should use `Parallelism Execution` instead of `Execution Framework`.

Reply:
This section is named by @XuHuaiyu , what do you want to express at this section?

Reply:
1. We can rename it as `Parallel Execution Framework`.
2. We'd better add some examples or figures to make the description easier to be undertood. e.g. The difference of the implementation between intra-parallelism and inter-parallelism.